ServiceDocumentSampleComponents/force-app/main/default/classes/FireCreateServiceDocumentInvocableAction.cls

```diff
@@ -0,0 +1,64 @@
+public with sharing class FireCreateServiceDocumentInvocableAction {
+
+    private static final String API_VERSION = 'v58.0';
+
+    private static final String CREATE_SERVICE_DOCUMENT_ACTION_PATH = '/services/data/'+ API_VERSION +'/actions/standard/createServiceDocument';
+
+    private static final String AUTHORIZATION_HEADER = 'Authorization';
+    private static final String AUTHORIZATION_BEARER_PREFIX = 'Bearer ';
+
+    private static final String CONTENT_TYPE_HEADER = 'Content-Type';
+    private static final String CONTENT_TYPE_JSON = 'application/json';
+
+    private static final String ACCEPT_HEADER = 'Accept';
+
+    /**
+     * RecordId: RecordId to generate service document for. WO/WOLI/SA id
+     * TemplateId: Service document template to generate document for. Starts with '0M0'
+     * Locale: (Optional) Locale to generate the document in. Example valid formats: 'ru', 'ru_PL'
+     * Title: (Optional) Used to name the document saved
+     */
+    @future (callout=true)
+    public static void TriggerCreateServiceDocumentInvocableAction(String recordId, String templateId, String locale, String title) {
+
+        System.debug('Attempting to queue Create Service Document');
```

Comment:
I think we can add debug statements as comments, so customers could choose to add or not by themself.

Reply:
I'm OK with that,